ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/textinput.html

```diff
@@ -37,7 +37,7 @@
               readonly="${text.readOnly}"
               required="${text.required}"
               placeholder="${text.placeHolder}"
-              data-cmp-required-message="${text.constraintMessages.required}">${text.default}</textarea>
+                  data-cmp-required-message="${text.constraintMessages.required}">${text.default}</textarea>
```

Comment:
I think this got in by mistake.Lets revert this change.

Reply:
yeah right, this is done